__tests__/workers/insertMissingAlerts.ts

```diff
@@ -0,0 +1,42 @@
+import { DataSource, In } from 'typeorm';
+import createOrGetConnection from '../../src/db';
+import { Alerts, User } from '../../src/entity';
+import {
+  GraphQLTestingState,
+  MockContext,
+  disposeGraphQLTesting,
+  initializeGraphQLTesting,
+  saveFixtures,
+} from '../helpers';
+import { usersFixture } from '../fixture/user';
+import { insertMissingAlerts } from '../../bin/insertMissingAlerts';
+
+let con: DataSource;
+let state: GraphQLTestingState;
+
+beforeAll(async () => {
+  con = await createOrGetConnection();
+  state = await initializeGraphQLTesting(() => new MockContext(con));
+});
+
+beforeEach(async () => {
+  jest.clearAllMocks();
+  await saveFixtures(con, User, usersFixture);
+});
+
+afterAll(() => disposeGraphQLTesting(state));
+
+describe('insert missing alerts script', () => {
```

Comment:
It's not really a worker, and ideally it's a once-off.
Not urgent, but technically might be better to remove this as it will just add unnecessary time to tests.

Reply:
Yeah, we can do that.